### DISCUSSION TITLE
gtf file on web browser vs desktop version

### DISCUSSION THREAD

**Discussion by koradiyajeel** [GMOD/jbrowse-components#4118] [Q&A] [answered]
Has anybody had issue with viewing signals from gtf file on web server vs desktop jbrowse2 version. I have uploaded the same .gtf file on web and desktop but only the desktop version shows signals in there to me

**Comment by koradiyajeel** [comment]
![image (4)](https://github.com/GMOD/jbrowse-components/assets/89552548/642c75aa-682e-47e0-a934-0f6be0525f67)
![image (5)](https://github.com/GMOD/jbrowse-components/assets/89552548/0800b5dc-d246-4250-aac3-9199a0c50f16)

**Reply by cmdcolin** [reply]
Hi there,
If you look at your gtf file, the first column is the "ref name"

I am not sure how you set up your web version, but you generally want the "refname" from the GTF to match the refname from the FASTA file -OR- you want to configure ref name aliases to allow different versions of the refname e.g. chr1 vs 1 to be allowed. The desktop version has ref name aliases configured, so it allows chr1 or 1 or even the ncbi style ID's  e.g. NC_000001.11 (it also has cytobands configured, hence the 'ideogram' in the overview bar)

**Reply by cmdcolin** [reply, marked as answer]
here is an example config section for the hg38 assembly that is basically the same as what is used on desktop . you can hand-edit your config.json to copy this (it can be a little tricky to setup all the command line flags for add-assembly to allow it)

https://github.com/GMOD/jbrowse-components/blob/42da823976e9fa1cfec41863b1472e2ed0721001/test_data/config.json#L49-L88

**Reply by koradiyajeel** [reply]
Hello, 

Thank you so much for the response it resolved the issue and helped me understand this better, Thanks again.

**Comment by koradiyajeel** [comment]
![image](https://github.com/GMOD/jbrowse-components/assets/89552548/1e0ce53f-49a9-4180-8b30-74af38fbd570)
on the same track, I created a .paf file using minimap2 with hg38 and monkey genome .fa file. when i try to load the synteny track suing the .paf file the gene which I want to look at is not able to find in monkey, would it be the same "refAliases" issue as above ? @cmdcolin

**Reply by cmdcolin** [reply]
I will have to double check refNameAliases fucntionality with PAF files. In my recollection, it might only work in "one direction". We might be able to fix that, but the best thing in that case is to have the names from the PAF exactly match the names from the assembly without any refNameAliases
 Random add-on: I am working on a new file format that aims to be more efficient than PAF for large e.g. eurkaryote-to-eukaryote genome alignments. It will only load a subset of the PAF, so if your PAF is large (the mouse-to-human alignment i experimented with was about 60MB gzipped) then the new file format called PIF only downloads ~3MB when you are viewing a small region instead of the entire file

**Reply by koradiyajeel** [reply]
yup names from the PAF exactly match the names from the assembly(monkey assembly). I have .paf file of hg38 and macaca_fascicularis and using hg38 and macaca_facicualris assemblies while loading the synteny track. now where I am having issue is when I navigate to gene of interest and say "link views" nothing appears.
sharing the link to my jbrowse session it has all the tracks 
what i am trying my session to look like is this example ensemble page - 
![image](https://github.com/GMOD/jbrowse-components/assets/89552548/463418a1-0300-449e-8c97-1813c2f1df32)

my session-
http://10.200.21.253:36888/?session=share-SZ2YuC9MP6&password=Z6QRh
